pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1627,7 +1623,7 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
    * @param permissions The file permissions.
    */
   async writeFile(filePath: string, fileContents: string, permissions: fs.Mode = 0o644) {
-    const workdir = await fs.promises.mkdtemp(path.join(os.tmpdir(), `rd-${ path.basename(filePath) }-`));
+    const workdir = await fs.promises.mkdtemp(path.join(os.tmpdir(), `rd-${ path.basename(filePath) }`));
```

Comment:
`mkdtemp()` takes a prefix; the `-` at the end is to make it clearer which part is the constant and which part is random.